eng/testing/tests.props

```diff
@@ -29,8 +29,6 @@
 
     <RuntimeIdentifier>$(PackageRID)</RuntimeIdentifier>
     <SelfContained>true</SelfContained>
-    <EnableTargetingPackDownload>false</EnableTargetingPackDownload>
-    <PlatformManifestFile />
```

Comment:
Is PlatformManifestFile no longer set to a custom one?

Reply:
No we don't set PlatformManifestFile anymore ourselves anywhere as we now use the live built TargetingPack which contains the PlatformManifest file and the SDK handles all the plumbing, same as in a customer app :)